index.html

```diff
@@ -1,12 +1,322 @@
+      <div
+        class=" container d-none d-md-flex flex-md-row justify-content-md-end gap-3 p-md-1"
+      >
+        <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+        <a href="#"><i class="fa-brands fa-twitter"></i></a>
+        <a href="#">English</a>
+        <a href="#">My page</a>
+        <a href="#" class="logout me-2">logout</a>
+      </div>
+    </header>
+    <nav class="navbar navbar-expand-md container">
+      <div class="container-fluid">
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent"
+          aria-controls="navbarSupportedContent"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+          id="button"
+        >
+          <span class="navbar-toggler-icon" id="open"></span>
+          <span
+            class="btn-close d-none navbar-toggler-icon Close"
+            id="close"
+          ></span>
+        </button>
+        <a class="navbar-brand ms-lg-5" href="index.html"
+          ><img src="images/Ccsummit2015_logo.png" alt="logo" class="logo"
+        /></a>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul
+            class="navbar-nav gap-3 mb-2  mb-lg-0 w-100 justify-content-between align-items-center "
+          >
+            <li class="nav-item ms-md-auto">
+              <a class="nav-link active" aria-current="page" href="index.html"
+                >Home</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#prog">Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#join">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link " href="#partners">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">News</a>
+            </li>
+
+            <li class="nav-item ms-md-auto">
+              <a href="#" class="nav-link campaign">More programs</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <main class="container-fluid px-0">
+      <section
+        class="intro container-fluid "
+      >
+      <div class="container d-flex flex-column gap-4 p-md-5 p-2 p-lg-5">
+        <div>
+          <h3 class="heading-small">"Hello, Sharing World"</h3>
+          <h1 class="heading-big">CREATIVE COMMONS <br>GLOBAL SUMMIT<br> 2015</h1>
+        </div>
+        <p class="intro-text">
+          Welcome to our 2022 web development Boot camp. The only course you
+          need to learn web development - HTML, CSS, JS, Node, and More!
+        </p>
+
+        <div class="event-date">
+          <h3 class="date">Starting on the (THU)15.10.2022 ~ 16(FRI)</h3>
```

Comment:
## Optional:
- Please you need to increase the font size. Your page will be more attractive if you follow the design(size) provided by Cindy Shin(The author of the original design). I would like to emphasize that this project can land you in a new career as Frontend Dev. Kindly take this Optional Serious
- 
![Capture](https://user-images.githubusercontent.com/63972670/190264220-eca18db7-9ce4-4dd9-8563-a67819e183b9.PNG)

- Your Design:
![Capture](https://user-images.githubusercontent.com/63972670/190264016-70ff38fc-3b2f-44ab-a8bb-9b33a623040e.PNG)